README.md

```diff
@@ -685,53 +669,54 @@ nock('http://my.server.com')
   .reply(200, '<html></html>')
 ```
 
-`delay()` could also be used as
+`delay(1000)` is an alias for `delayConnection(1000).delayBody(0)`  
+`delay({ head: 1000, body: 2000 })` is an alias for `delayConnection(1000).delayBody(2000)`  
+Both of which are covered in detail below.
 
-```
-delay({
-   head: headDelayInMs,
-   body: bodyDelayInMs
-})
-```
+#### Delay the connection
+
+You are able to specify the number of milliseconds that your connection should be idle before it starts to receive the response.
 
-for example
+To simulate a socket timeout, provide a larger value than the timeout setting on the request.
 
 ```js
 nock('http://my.server.com')
   .get('/')
-  .delay({
-    head: 2000, // header will be delayed for 2 seconds, i.e. the whole response will be delayed for 2 seconds.
-    body: 3000, // body will be delayed for another 3 seconds after header is sent out.
-  })
+  .delayConnection(2000) // 2 seconds
   .reply(200, '<html></html>')
+
+req = http.request('http://my.server.com', { timeout: 1000 })
 ```
 
-### Delay the connection
+While all of Nock's delay functions will delay real clock time for the request/response lifetime,
+`timeout` events are emitted almost immediately by comparing the requested connection delay to the timeout parameter passed to `http.request()` or `http.ClientRequest#setTimeout()`.  
+This allows you to test timeouts without using fake timers or slowing down your tests.
```

Comment:
These two things feel like contradictory:

- "While all of Nock's delay functions will delay real clock time for the request/response lifetime"
- "This allows you to test timeouts without using fake timers or slowing down your tests."

How is it that they're both true?

Reply:
>How is it that they're both true?

Because a `timeout` event does not directly affect the request or response.
Maybe "lifetime" is the wrong wording.

The timeout event is emitted "immediately", if the client chooses to not take an action (eg abort the request) the request and response will continue on as normal, after real time as passed.
Which is identical behavior to how Nock behaves now when a caller uses `delayConnection` and sets a timeout on the request (without `socketDelay`).  

As for not slowing down a users tests, this assumes their code or client _does_ take action on the `timeout` event and subsequently the test can finish quick.

Reply:
Ahhh I see. I think putting the explanation of real clock time at the end would make it clearer. How about something like this:

> Nock emits `timeout` events almost immediately by comparing [...] This allows you to test timeouts without using fake timers or slowing down your tests. If the client chooses to not take an action (e.g. abort the request), the request and response will continue on as normal, after real clock time has passed.